src/economy/economy.cpp

```diff
@@ -1028,6 +1028,7 @@ void rebalance_needs_weights(sys::state& state, dcon::market_id n) {
 				}
 				auto& w = state.world.market_get_life_needs_weights(n, c);
 				w = ideal_weighting * state.defines.alice_need_drift_speed + w * (1.0f - state.defines.alice_need_drift_speed);
+				w = 1.f;
```

Comment:
Did you mean to write this? It seems to completely discard the previous calculation. If you want to keep the old calculation around, I suggest commenting it out so that the linter won't complain

Reply:
Yes, the plan was to discard it. Will comment out old calculation.